### ISSUE TITLE
lsad=0 crash

### ISSUE THREAD

**Issue by realfinder** [pinterf/mvtools#49]
as said here https://forum.doom9.org/showthread.php?p=1954328#post1954328

![image](https://user-images.githubusercontent.com/2512153/138885559-255818c5-41d5-4b65-bdd5-f907afbced28.png)

```
AVISource("mv test.avi")
super_search = MSuper(hpad=16,vpad=16)

bv1 = super_search.MAnalyse(isb = true, delta = 1, overlap= 4, lsad=0)

MMask(last, bv1)
```

**Comment by pinterf** [comment]
Fixed (in my desktop) by prechecking lsad- when it is zero then the resultin lambda is zero as well.
Note: vs-mvtools is no different there, no guard. Some time ago they changed int64 calculation to 'double' arithmetic.
But there division by zero with 'double' types does not crash, calculation goes on with a "NaN" value
https://github.com/dubhater/vapoursynth-mvtools/blob/master/src/PlaneOfBlocks.cpp#L461
Result (in VS-mvtools) Instead of zero they put -2147483648 to lambda

**Comment by pinterf** [comment]
Fixed in b0e927c2e3ab984f1ceaba1751f9551954c36351
